src/docbkx/chapter13-getting-started-monitor.xml

```diff
@@ -30,14 +30,45 @@
 		</para>
 		<mediaobject>
 			<imageobject>
-				<imagedata fileref="monitor_01_login.jpg" format="JPG"
-					scalefit="1" />
+				<imagedata fileref="monitor_01_login.jpg" format="JPG" scalefit="1"/>
 			</imageobject>
 		</mediaobject>
 		<para>In the community edition of DataCleaner, you will find
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during
+				installation as<emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using
```

Comment:
Isn't it C:\Users\USER_NAME?

And I only ever use English Windows, so I'm not sure, but I guess they still translate folder names for localized Windows versions? Maybe rephrase into something like "A `.datacleaner` folder is created in your user home by the installer,  in an English Windows 7 this would probably be `C:\Users\YOUR_USER_NAME\.datacleaner`"